js/src/workspaces/window.js

```diff
@@ -766,11 +766,11 @@
                                  '</ul>',
                                  '</a>',
                                  '{{#if ThumbnailsView}}',
-                                 '<a href="javascript:;" class="mirador-btn mirador-icon-thumbs-view thumbnails-option"><i class="fa fa-th fa-lg fa-rotate-90 fa-fw"></i>',
+                                 '<a href="javascript:;" class="mirador-btn mirador-icon-thumbs-view thumbnails-option" role="button" aria-label="Change to Thumbnails Mode"><i class="fa fa-th fa-lg fa-rotate-90 fa-fw"></i>',
                                  '</a>',
                                  '{{/if}}',
                                  '{{#if MetadataView}}',
-                                 '<a href="javascript:;" class="mirador-btn mirador-icon-metadata-view" title="{{t "objectMetadata"}}"><i class="fa fa-info-circle fa-lg fa-fw"></i></a>',
+                                 '<a href="javascript:;" class="mirador-btn mirador-icon-metadata-view" title="{{t "objectMetadata"}}" role="button" aria-label="View Information/Metadata about Image"><i class="fa fa-info-circle fa-lg fa-fw"></i></a>',
```

Comment:
Can you change this to "View Information/Metadata about Object"?  There will eventually be another info button that will allow users to view information about a canvas, so using Image in this description will be confusing.  And, that info button refers to Object-level metadata.